### DISCUSSION TITLE
Can't use bottles in China

### DISCUSSION THREAD

**Discussion by lumingzh** [bottlesdevs/Bottles#575] [Q&A]
The repository domain name "raw.githubusercontent.com" is banned in China, is there any other way to install components/runners/installers?
BTW using bottles in flatpak way.

**Comment by mirkobrombin** [comment]
Is the ban only on the domain or also the ip?

**Reply by lumingzh** [reply]
I'm not sure. The great firewall of Chinese internet is very complicate. I heard this site of github is using amazon cloud service, I think it's about amazon cloud, not against github itself.

**Reply by mirkobrombin** [reply]
In that case I don't know how to do it because Bottles is heavily dependent on GitHub. It would take someone with the effort to maintain a Chinese repo

**Reply by lumingzh** [reply]
Is there any possible to maintain a gitlab mirror? Gitlab can be accessed normally in China.

**Reply by mirkobrombin** [reply]
It is doable but I don't have the time to devote at the moment. Is it a problem to use a VPN until I have time for a solution?

**Reply by lumingzh** [reply]
Take your time, it's ok for me. I wish you can find a good solution. Thank you anyway.
